modules/ztp-sno-node-reboot-scenarios.adoc

```diff
@@ -4,20 +4,20 @@
 
```

Comment:
in line 3 it is mentioned the old place of this module (// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc), let's update that to the new place: 
"nodes/nodes/nodes-nodes-working.adoc"